packages/hack_parallel/hack_parallel.1.0.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Rijnard van Tonder <rvantonder@gmail.com>"
+authors: "Facebook. Modifications by Rijnard van Tonder"
+homepage: "https://github.com/rvantonder/hack_parallel"
+bug-reports: "https://github.com/rvantonder/hack_parallel/issues"
+dev-repo: "git+https://github.com/rvantonder/hack_parallel.git"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs "@install"]
+]
+depends: [
+  "conf-sqlite3"
+  "core_kernel" {< "v0.14"}
+  "ocaml" {>= "4.04.1"}
+  "ppx_deriving"
+  "ppxlib" {< "0.9.0"}
+  "sexplib" {< "v0.14"}
+]
+depexts: [
+  "pkg-config"
```

Comment:
These depext's aren't quite the right format: you can just depend on `conf-pkg-config` above in the `depends` field  and drop the `depexts` clause entirely. See the `conf-sqlite3` and `conf-pkg-config` opam files for the opam 2.0 depext format if curious.

Reply:
Handy, thanks! Updated.